### PR TITLE
Feat livedraft event

### PR DESCRIPTION
**Pull request by rishi9684**
#Live Draft event tiles
#gql file for same
#added content for live draft event
#pattern images cdn link added

### CODE REVIEW

File: config/templates/grids/livedraft-grid/body.vue [hunk 1]
```diff
@@ -0,0 +1,300 @@
...
+        </v-col>
+      </v-row>
+    </div>
+  </v-flex>
+</template>
+
+<script>
+import nuxtconfig from '~/nuxt.config'
+export default {
+  props: ['items', 'context'],
```
Comment (SukantGujar) [inline config/templates/grids/livedraft-grid/body.vue:246]: Give prop types

**Review by SukantGujar — CHANGES_REQUESTED** [review]

File: config/templates/grids/livedraft-grid/body.vue (hunk 1 above)
Comment (rishi9684) [reply config/templates/grids/livedraft-grid/body.vue]: added type

File: nuxt.config.js [hunk 2]
```diff
@@ -109,7 +113,8 @@ export default {
   axios: {
     apiEndpoint: '/svc/api/',
     backendBaseUrl: process.env.PUBLIC_DOMAIN || '',
-    baseURL: `https://${process.env.PUBLIC_DOMAIN}${basePath}`,
+    // baseURL: `https://${process.env.PUBLIC_DOMAIN}${basePath}`,
+    baseURL: `http://localhost:3000${basePath}`,
```
Comment (SukantGujar) [inline nuxt.config.js:117]: Are you sure? This should come from env.

**Review by SukantGujar — CHANGES_REQUESTED** [review]

File: nuxt.config.js (hunk 2 above)
Comment (rishi9684) [reply nuxt.config.js]: removed loaclhost url

**Review by SukantGujar — APPROVED** [review]

File: nuxt.config.js (hunk 2 above)
Comment (rishi9684) [reply nuxt.config.js]: >>Are you sure? This should come from env.
i had just added baseURL: `http://localhost:3000${basePath}`, for runing on local only

**Pull request merged**